docroot/sites/default/default.services.yml

```diff
@@ -36,6 +36,22 @@ parameters:
     # @default none
     # cookie_domain: '.example.com'
     #
+    # Set the session ID string length. The length can be between 22 to 256. The
```

Comment:
here and below - these are updates from scaffolding files. These files are preserved here for refrerence